subsys/net/ip/Kconfig.mgmt

```diff
@@ -51,6 +51,24 @@ config NET_MGMT_EVENT_INFO
 	  and listeners will then be able to get it. Such information depends
 	  on the type of event.
 
+config NET_MGMT_EVENT_MONITOR
```

Comment:
Shall these new configs be added to https://github.com/zephyrproject-rtos/zephyr/blob/master/tests/net/all/prj.conf, so that this new feature is built by the CI?

Reply:
Good point, I will add that.